client/components/site-selector/index.jsx

```diff
@@ -297,6 +297,8 @@ export default React.createClass( {
 			'is-single': user.get().visible_site_count === 1
 		} );
 
+		const hiddenSitesCount = user.get().site_count - user.get().visible_site_count;
+
 		return (
 			<PreferencesData>
```

Comment:
Huh, totally unrelated, but is `<PreferencesData>` even doing anything here? It'd be passing a `preferences` prop to its child, which is... `div`.